tests/cases/unittests/jsDocParsing.ts

```diff
@@ -998,7 +1001,10 @@ module ts {
                         ? JSON.parse(Utils.sourceFileToJSON(v))
                         : v;
                 }, 4);
-                
+
+                // Remove the parserContextFlags from the comparison
+                result = result.replace(/\,\n\s+\"parserContextFlags\": \"JavaScriptFile\"/g, "");

```

Comment:
Because the harness cares about what flags the parser sets on the AST (see https://github.com/Microsoft/TypeScript/blob/master/src/harness/harness.ts#L277 ), but I don't care to go edit the 100s of stringified AST nodes in this file now we correctly consider this flag "*ParserGenerated*" - especially as 1). It's not meaningful for what these tests are validating, and 2). There's been talk of potentially changing the contextFlags, and I'd hate to have to edit them all again shortly down the road.